gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/Client.java

```diff
@@ -642,7 +642,7 @@ public static class AliasClusteredClient extends Client {
 
         @Override
         public CompletableFuture<ResultSet> submitAsync(final Bytecode bytecode) {
-            return submitAsync(bytecode, RequestOptions.EMPTY);
+            return submitAsync(bytecode, DriverRemoteConnection.getRequestOptions(bytecode));
```

Comment:
I wonder if the "getRequestOptions" method should be extracted out of the DRC class now that it's also being used by the Client. It feels weird to have Client depend on DRC...

Reply:
Moved `getRequestOptions` to a new static subclass `ClientUtil` in `Client.java` to remove dependency.